addons/website_sale/views/templates.xml

```diff
@@ -245,7 +245,8 @@
                                             <t t-foreach="tr_product" t-as="td_product">
                                                 <t t-if="td_product">
                                                     <t t-set="product" t-value="td_product['product']" />
-                                                    <td t-att-colspan="td_product['x'] != 1 and td_product['x']"
+                                                    <td id="product_item"
```

Comment:
Now you have a DOM with the same id like 50 times...
To target an element with a xpath:
- either the element is unique and you can use an id
- otherwise: use a class or data-name or simply the DOM when it is safe enough.

Here targeting the `td` in the right `foreach` or the `oe_product` class are both safe options.

Reply:
Thx